core/arch/arm/kernel/abort.c

```diff
@@ -555,6 +560,8 @@ void abort_handler(uint32_t abort_type, struct thread_abort_regs *regs)
 
 	switch (get_fault_type(&ai)) {
 	case FAULT_TYPE_IGNORE:
+		/* Allow platform-specific handling */
+		plat_abort_handler(regs);
```

Comment:
I'm not so keen on adding hooks that the compiler can't optimize away if unused.
How about using a config variable so the function call can be optimized away if not needed?

Reply:
Ok for using a config switch, how about `CFG_EXTERNAL_ABORT_PLAT_HANDLER`,